Library/Dnn.PersonaBar.UI/admin/personaBar/scripts/permissionGrid.js

```diff
@@ -361,10 +361,16 @@ if (typeof dnn.controls === "undefined" || dnn.controls === null) { dnn.controls
                 return false;
             }
 
+            var defaultPermission = this.data.permissionDefinitions.find(
+                function(permission) {
+                    return permission.permissionId == 5;

```

Comment:
Please change so that we don't have a magical `5` here. At least let's introduce a friendly-named constant

Reply:
I will introduce a permissions map.